packages/core/src/extensions/extension-discovery/stop-watching-extensions-on-quit.injectable.ts

```diff
@@ -3,23 +3,19 @@
  * Licensed under MIT License. See LICENSE in root directory for more information.
  */
 import { getInjectable } from "@ogre-tools/injectable";
-import { beforeQuitOfBackEndInjectionToken } from "../../main/start-main-application/runnable-tokens/before-quit-of-back-end-injection-token";
+import { beforeQuitOfBackEndInjectionToken } from "../../main/start-main-application/runnable-tokens/phases";
 import extensionDiscoveryInjectable from "./extension-discovery.injectable";
 
 const stopWatchingExtensionsOnQuitInjectable = getInjectable({
   id: "stop-watching-extensions-on-quit",
 
-  instantiate: (di) => {
-    const extensionDiscovery = di.inject(extensionDiscoveryInjectable);
+  instantiate: (di) => ({
+    run: async () => {
+      const extensionDiscovery = di.inject(extensionDiscoveryInjectable);
```

Comment:
Do you mean the 6.4 version?

If you are talking about the move of the `di.inject(...)` call, then it doesn't matter that much. It is more of style thing. I think that this is better (and less of a footgun) since if any of the `di.inject()` calls rely on some runnable finishing then this is where they have to be.

I prefer just putting all `di.inject(...)`'s inside the `run` as a rule of thumb. They are less spread out. We have less nesting and indentation.